source/examples/rapids-azureml-hpo/notebook.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>Could you put this in a <code>docref</code> directive so that it is clear it is a reference to another part of the deployment docs?</p><p> </p><p>See the "Prerequisites" section of this notebook for an example https://github.com/rapidsai/deployment/blob/main/source/examples/xgboost-gpu-hpo-job-parallel-ngc/notebook.ipynb</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>